diagnostics/model_test/verify_model.py

```diff
@@ -2,10 +2,38 @@
 import argparse
 import numpy as np
 import ngraph
+from google.protobuf import text_format
 import json
 import os
 
 
+def createFolder(directory):
+    try:
+        if not os.path.exists(directory):
+            os.makedirs(directory)
+    except OSError:
+        print('Error: Creating directory. ' + directory)
+
+
+def set_os_env(select_device):
+    if select_device == 'CPU':
+        # run on TF only
+        ngraph.disable()
+        os.environ['NGRAPH_TF_BACKEND'] = 'CPU'
+    else:
```

Comment:
I think we don't need `os.environ['NGRAPH_TF_BACKEND'] = 'CPU'` when ngraph is disabled

Reply:
Will remove it.